content/actions/using-workflows/workflow-syntax-for-github-actions.md

```diff
@@ -157,6 +157,42 @@ jobs:
 ```
 {% endraw %}
 
+{% if actions-inherit-secrets-reusable-workflows %}
+
+#### `on.workflow_call.secrets.inherit`
+
+Use the `inherit` keyword to pass all the calling workflow's secrets to the called workflow. This includes all secrets the calling workflow has access to, and also includes organization, repository, and environment secrets. The `inherit` keyword can also be used to pass secrets across repositories within the same organization, or across organizations within the same enterprise.
```

Comment:
@martin389 I am not sure if we need "also" there as the secrets indicate these 3 types of secrets. 
```suggestion
Use the `inherit` keyword to pass all the calling workflow's secrets to the called workflow. This includes all secrets the calling workflow has access to, that is, the organization, repository, and environment secrets that can be accessed from the calling worklfow. The `inherit` keyword can also be used to pass secrets across repositories within the same organization, or across organizations within the same enterprise.
```

Reply:
I've updated this, I believe `namely` is the best way to phrase this.